kf_model_fhir/mappers/resources/phenotype.py

```diff
@@ -0,0 +1,84 @@
+"""
+This module converts Kids First phenotypes to FHIR phenotypes (derived from FHIR Observation).
+"""
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+
+from common.utils import make_select, get, make_identifier
+
+
+# ttps://www.hl7.org/fhir/valueset-observation-status.html
+status = {
+    constants.COMMON.TRUE: 'registered'
+    constants.COMMON.FALSE: 'unknwon'
+}
+
+# https://www.hl7.org/fhir/valueset-observation-interpretation.html
+interpretation = {
+    constants.PHENOTYPE.OBSERVED.NO: {
+        'system': 'http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation',
+        'code': 'NEG',
+        'display': 'Negative'
+    },
+    constants.PHENOTYPE.OBSERVED.YES: {
+        'system': 'http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation',
+        'code': 'POS',
+        'display': 'Positive'
+    }
+}
+
+def yield_participants(eng, table, study_id, participants):
```

Comment:
not participants

Reply:
Renamed:

- phenotype.py >> kfdrc_phenotype.py
- yield_participants >> yield_kfdrc_phenotype